code/modules/games/cas.dm

```diff
@@ -28,6 +28,7 @@
 	card_text_file = "strings/cas_black.txt"
 
 /obj/item/toy/cards/deck/cas/Initialize()
+	. = ..()
```

Comment:
No side effects from this one?

Reply:
There shouldn't be any, the parent is all the way down in /obj/item/Initialize() and it doesn't do anything funky, and neither does this proc really.